test/punycode_test.c

```diff
@@ -211,9 +200,57 @@ static int test_puny_overrun(void)
     return 1;
 }
 
+static int test_dotty(void)
```

Comment:
Nit: `dotty` makes one think it has something with terminal devices. Could you please choose a different name?

Reply:
Sigh....